src/configParamsJSON/sidebarConfigParams.json

```diff
@@ -618,7 +618,7 @@
         },
         {
           "type": "subsection",
-          "id": "share",
+          "id": "screenshot",
```

Comment:
looks like the subsection id was changed from `share` to `screenshot`. Was this intended?

Reply:
Oops good catch. I'll fix that one